bp-loader.php

```diff
@@ -19,20 +19,20 @@
  * License URI:       https://www.gnu.org/licenses/gpl-2.0.html
  * Text Domain:       buddypress
  * Domain Path:       /bp-languages/
- * Requires PHP:      5.6
- * Requires at least: 6.1
+ * Requires PHP:      7.0
```

Comment:
According to [WordPress stats](https://wordpress.org/about/stats/), PHP 7.0 has less than 2% of usage. PHP 7.3 has around ~3.8%.

And PHP 7.4, 37.6%. **I personally would like to drop support for <7.3**. Meaning that BuddyPress would require 7.4>.

It's not clear to me why we were supporting 5.6, but we don't have data specific to BuddyPress, as far as I know.

Reply:
7.0 is the minumum version supported by 6.4, that's why I defaulted to 7.0.

Reply:
I believe we should follow WordPress requirements about this. Raising PHP version according to the minimum required WP version: WP 6.4 => PHP 7.0 is fine with me.
WP 6.6 is supporting back to [7.2](https://core.trac.wordpress.org/browser/branches/6.6/src/wp-includes/version.php)